### PR TITLE
Gives crabs their own meat + recipes

### PR DESCRIPTION
**Pull request by TheRealGLH**
## About The Pull Request

I noticed a while ago that crabs, when gibbed, just dropped generic slabs of meat and felt as if they could do with their own meat.
If there is some famous crab dish that you feel should be added, feel free to tell me.

## Why It's Good For The Game

Little details like this are neat and make cooking fun.

## Changelog
:cl: giygaslittlehelper
tweak: Crabs now drop their own meat rather than generic meat slabs, which can be used in a couple of new recipes.
/:cl:

### CODE REVIEW

**Comment by ShizCalev** [comment]
Yum

**Comment by Farquaar** [comment]
Mmm, crab bisque
Maybe one day we'll be able to make sushi that isn't poison too?

**Comment by tralezab** [comment]
Where's the imitation crab?

**Comment by plapatin** [comment]
crunchy crab pizza when

File: code/modules/food_and_drinks/food/snacks_soup.dm [hunk 1]
```diff
@@ -225,3 +225,11 @@
 	bonus_reagents = list("nutriment" = 4, "vitamin" = 6)
 	tastes = list("beet" = 1)
 	foodtype = VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/soup/bisque
+	name = "bisque"
+	desc = "A classic entr�e from France."
```
Comment (MrDoomBringer) [inline code/modules/food_and_drinks/food/snacks_soup.dm]:
```suggestion
	desc = "A classic entrée from Space-France."
```

**Review by MrDoomBringer — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/food/snacks_soup.dm (hunk 1 above)
Comment (optimumtact) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: byond has limited support for unicode right now, don't use fancy letters please

Comment (nicbn) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: You could use &eacute; (maybe (probably not))

Comment (optimumtact) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: just avoid them all together until proper unicode support lands

**Comment by HomocidlePancakes** [comment]
Can you add duck meat while you are at it?

**Comment by Grazyn** [comment]
Crab cake, obviously
Crab roll (should be lobster roll but eh)
Crab risotto
Crab salad (it's a thing)

**Comment by KrabSpider** [comment]
Crab rangoon, add it. No balls.

**Comment by OverlyGeneric** [comment]
> I don't think there is a duck mob in the code, is there any codebase where I would be able to use the sprites from that also has ducks? You know, for uniformity.
> 
> Would it be atomic to even add ducks in this PR, or should I do them separately?

We already have geese and so why not ducks? I recommend that you PR it separately.

I also suggest adding crab icecream and Krabby Patties because _it's not like we have enough references_ anyway.

**Comment by MrDoomBringer** [comment]
@OverlyGeneric Krabby Patties are already referenced in this pr

```
	/obj/item/reagent_containers/food/snacks/burger/crab
	name = "crab burger"
	desc = "A delicious patty of the crabby kind, slapped in between a bun."
```

**Comment by Bawhoppen** [comment]
@TheRealGLH I think I've seen ducks before. Not sure what codebase. Try Hippie, Para, Yog... Pretty sure if it's not one of those, it's some /tg/ downstream.

**Comment by HomocidlePancakes** [comment]
My dum brain messed up ducks and geese. I meant to say "Can you add goose meat ehile you are at it?"

File: code/modules/food_and_drinks/food/snacks_soup.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: De-fancy this

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Comment by MrDoomBringer** [comment]
this pr was closed by: é gang

**Comment by ExcessiveUseOfCobblestone** [comment]
imagine losing to a webedit

**Pull request closed without merging**